controllers/drpolicy_controller.go

```diff
@@ -156,6 +163,44 @@ type objectUpdater struct {
 	log    logr.Logger
 }
 
+func (u *objectUpdater) clusterFenceHandle() error {
+	if u.object.Status.DRClusters == nil {
+		u.object.Status.DRClusters = make(map[string]ramen.ClusterStatus)
+	}
+
+	for _, managedCluster := range u.object.Spec.DRClusterSet {
+		// TODO:
+		// 1) For now by default fenceStatus is ClusterFenceStateUnfenced.
+		//    However, we need to handle explicit unfencing operation to unfence
+		//    a fenced cluster below, by deleting the fencing CR created by
+		//    ramen.
+		//
+		// 2) How to differentiate between ClusterFenceStateUnfenced being
+		//    set because a manually fenced cluster is manually unfenced against the
+		//    requirement to unfence a cluster that has been fenced by ramen.
+		//
+		// 3) Handle Ramen driven fencing here
+		if managedCluster.ClusterFence == ramen.ClusterFenceStateUnfenced ||
```

Comment:
These two if blocks can be changed to simpler  something like this.  
clusterStatus := ramen.ClusterStatus{
                
                        Name:    managedCluster.Name,  
                         Status:  ramen.ClusterFenced
        }